app/models/account.rb

```diff
@@ -0,0 +1,14 @@
+class Account < ApplicationRecord
+  belongs_to :group
+  belongs_to :owner, class_name: 'User', foreign_key: 'owner_id'
```

Comment:
I think we don't need this `owner` because this account belongs to a group that already has an owner.

So, if needed, we can add something like `delegate :owner, to: :group` and we have the account owner.

For now, this should be good enough but we can improve our permissions in the future. eg: [CanCanCan](https://github.com/CanCanCommunity/cancancan)